pyproject.toml

```diff
@@ -55,20 +59,13 @@ testing = [
 ]
 pre_commit = [
     "pre-commit",
-    "pylint~=2.12.2",
-    "mypy==0.930",
+    "pylint~=3.3.1",
```

Comment:
I'm gonna get rid of pylint for ruff, but in a separate PR.

Reply:
I begin tu really appreciate the beauty of single rust binaries, this pylint pin was a reason why the installation for newer Python versions did not work. :-(